packages/components/src/focal-point-picker/index.js

```diff
@@ -27,7 +27,7 @@ export class FocalPointPicker extends Component {
 		this.state = {
 			isDragging: false,
 			bounds: {},
-			percentages: {},
+			percentages: FocalPointPicker.defaultProps.value,
```

Comment:
It's neither here nor there to the specific fix, but this looks to me like a classic violation of an previously-documented anti-pattern:

http://web.archive.org/web/20160430110308/http://facebook.github.io/react/tips/props-in-getInitialState-as-anti-pattern.html